src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -57,16 +55,6 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
         return commandResult;
     }
 
-    @Override
-    public ReadOnlyAddressBook getAddressBook() {
-        return model.getAddressBook();
-    }
-
-    @Override
-    public ObservableList<Person> getFilteredPersonList() {
-        return model.getFilteredPersonList();
-    }
-
     @Override
     public Path getAddressBookFilePath() {
```

Comment:
Should this be removed or it's waiting for model dependencies removal?

Reply:
oh if you are removing it I can add it back